pages/key/import.js

```diff
@@ -97,22 +103,44 @@ export default function ImportKey() {
                 style={{width: '100%'}}
               >
                 <FormLabel
+                  display={['none', 'inherit']}
                   htmlFor="key"
                   style={{color: 'white', fontSize: rem(13)}}
                 >
                   {t('Encrypted private key')}
                 </FormLabel>
-                <Flex width="100%" style={{marginBottom: rem(20)}}>
+                <Flex
+                  width="100%"
+                  style={{marginBottom: rem(20), position: 'relative'}}
+                >
                   <Input
                     id="key"
+                    size={size}
                     value={state.key}
                     borderColor="xblack.008"
                     backgroundColor="xblack.016"
                     onChange={e => setState({...state, key: e.target.value})}
                     placeholder={t('Enter your private key backup')}
                   />
+                  <Box
+                    display={['initial', 'none']}
+                    style={{
+                      ...borderRadius('right', rem(6)),
+                      cursor: 'pointer',
+                      fontSize: rem(20),
+                      position: 'absolute',
+                      top: rem(10),
+                      height: '100%',
+                      right: rem(6),
+                      zIndex: 5,
+                    }}
+                    onClick={() => {}}
+                  >
+                    <img src="/static/qr-scan-icn.svg" />
```

Comment:
Add icon to `shared/components/icons.js` and use here like `<QrIcon boxSize={5} />`